src/constants.ts

```diff
@@ -29,4 +29,9 @@ export const MESSAGES = {
         `We will attempt to install the latest Salesforce Code Analyzer plugin on your behalf.`,
     SCANNER_PLUGIN_INSTALL_FAILED: `Failed to install the latest Salesforce Code Analyzer plugin on your behalf.`
 }
+export const MESSAGE_FCNS = {
```

Comment:
What does FCNS stand for?

Reply:
FCN stands is the typical way to abbreviate "function", so FCNS means functions.
I was really sad to see that javascript doesn't have template strings... so there is no way to easily format strings off a common template. This is what was recommended everywhere on the internet. Not my favorite to be honest... but MESSAGE_FNCS as opposed to MESSAGES allows us to know that to get the message, you need to supply its arguments.